### DISCUSSION TITLE
No tags in scan mode 4.0.2

### DISCUSSION THREAD

**Discussion by om3wuc** [rtl-airband/RTLSDR-Airband#316] [General]
Just updated to 4.0.2, no channel tags in scan mode. No tags in player, no tags on icecast status page (currently playing).  Is the channel tagging working in 4.0.2 ?

**Comment by szpajder** [comment]
Yes, tags work in 4.0.2.

If they don't, then it might be a problem described in
https://github.com/szpajder/RTLSDR-Airband/issues/203 .

**Reply by om3wuc** [reply]
Maybe the libshout is the problem. I'm on bullseye, libshout 2.4.5, Pi Zero W. Unfortunately, I'm not able to downgrade libshout on this HW/SW.

**Reply by om3wuc** [reply]
Fixed! libshout 2.4.5 removed, compiled 2.4.1 from source and the tags are back.

**Reply by szpajder** [reply]
Metadata updates are broken in libshout 2.4.5 (see https://gitlab.xiph.org/xiph/icecast-libshout/-/issues/2328).
